google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerOptions.java

```diff
@@ -1155,6 +1158,15 @@ public Builder setEmulatorHost(String emulatorHost) {
       return this;
     }
 
+    /**
+     * Enable or disable leader aware routing. Leader aware routing would route all requests in
+     * RW/PDML transactions to the leader region.
+     */
+    public Builder setLeaderAwareRouting(boolean leaderAwareRoutingEnabled) {
```

Comment:
I think we should have a method for disabling leaderAwareRouting, and one for enabling leaderAwareRouting. As the default is true and we need a process for users to opt out of it, it would make more sense to have `disableLeaderAwareRouting()` method (which sets the flag to false), and a method enableLeaderAwareRouting(to set the flag back to true). This is to improve user experience with using `SpannerOptions`.

CC: @olavloite for any inputs on this.